src/style.css

```diff
@@ -1 +1,57 @@
 /* add styles here */
+html {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  position: relative;
+  width: 96px;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Although not directly visible in this snippet, ensure that your font-family declarations throughout your CSS include fallback options.